preshed/bloom.pyx

```diff
@@ -0,0 +1,79 @@
+# cython: infer_types=True
+# cython: cdivision=True
+#
+cimport cython
+
+from murmurhash.mrmr cimport hash64
+import math
+
+def optimal_params(members, error_rate):
```

Comment:
We probably want this to have a verbier name. Maybe `get_best_size_and_hash_count`? That also lets us return a tuple instead of a dict.